service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/AzureDatabaseUtilsRunnerTest.java

```diff
@@ -171,6 +174,8 @@ private void assertResults(Map<String, String> expectedEnv) {
     var expectedEnvWithCommon = new HashMap<>(expectedEnv);
     expectedEnvWithCommon.putAll(
         Map.of(
+            "AZURE_ENVIRONMENT",
+            AZURE_ENV,
```

Comment:
#1806 added a new env var, but did not update this expectation to include the new env var. This caused tests to fail. See https://github.com/DataBiosphere/terra-workspace-manager/pull/1806/files#diff-7e2975f5c6f927eade14d4008340e3ec0d6d7df12e5669d0b3e3cd3784f0cac3